### DISCUSSION TITLE
Welcome to every-breath-you-take Discussions!

### DISCUSSION THREAD

**Discussion by kieranabrennan** [kieranabrennan/every-breath-you-take#8] [Announcements]
👋 Welcome!
  We’re using Discussions as a place to engage with how others are using and building on the project. Please:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by AdamCraven** [comment]
Thank you for the awesome project, I bought a polar H10 off the back of seeing it on hacker news and have been using it throughout the day to train myself towards a better, more relaxed, HRV.

Some features that could be useful additions:

1) Audible noises for when the app is in the background. A slow gentle noise when your HRV is good and a faster noise when it's not so good (a weighted average, so the noises aren't changing all the time)
2) Outputting of logs. The app would have to become a lot more complex if it had to do all the data-analysis itself, but creating an output in JSON format, you could easily input it into a data analytics platform or tool like grafana, kibana, etc to track it over long periods of time to see patterns.

**Reply by kieranabrennan** [reply]
Hey @AdamCraven awesome to hear you're using it. Thanks for the suggestions, I've been thinking about adding the ability to save sessions and do longer term trend analysis, and love the idea of the audio feedback

**Comment by ericrascon** [comment]
Thank you for this! As someone who's taken an interest in HRV data earlier this year this couldn't be more helpful.

Question. When I get "Rejected low HRV value" in the terminal. Does that mean it's reading my biometric data correctly?

**Comment by leex** [comment]
It says firmware 5.0.0 is required, but my device is fully updated at version 3.2.0, is this correct?

**Comment by leex** [comment]
I'm using apple silicon on M2, the app opens then crashes with error:

Termination Reason:    Namespace TCC, Code 0 
This app has crashed because it attempted to access privacy-sensitive data without a usage description. The app's Info.plist must contain an NSBluetoothAlwaysUsageDescription key with a string value explaining to the user how the app uses this data.

**Reply by leex** [reply]
Nevermine, for those that follow, you'll need to do the pyinstaller and build the app for it to run with the correct bluetooth permissions.
